src/common/options/rgw.yaml.in

```diff
@@ -2429,7 +2429,7 @@ options:
 - name: rgw_sync_lease_period
   type: int
   level: dev
-  default: 2_min
+  default: 10_min
```

Comment:
i don't think there's any reason to change this lease duration. if a radosgw instance acquires a lock and crashes, we don't want to wait a full 10 minutes before another instance can reacquire its lock and resume sync